File: bot/commands/util/serverstats.js
exports.run = async (bot) => {
  bot.registerCommand("serverstats", async (message, args) => {
    await bot.checkUserAndGuild(message)
    if (bot.checkBannedUsers(message.author.id)) return

    let userStats
    if (!args[0]) {
      userStats = message.author
    } else if (message.mentions[0]) {
      userStats = message.mentions[0]
    } else {
      userStats = message.author
    }


    bot.database.Userdata.findOne({ userID: userStats.id }, async (err, userdata) => {
      if (err) bot.log("error", err)

      if (userdata && userdata.stats) {
        const statsEmbed = {
          embed: {
            author: {
              name: `${userStats.username}'s stats in ${message.channel.guild.name}`,
              icon_url: userStats.avatarURL
            },
            color: bot.getEmbedColor(bot, message),
            thumbnail: {
              url: message.channel.guild.iconURL
            },
            fields: [
              {
                name: "OwO Count",
                value: `\`${userdata.stats.guilds[message.channel.guild.id].owoCount}\``,
                inline: true
              },
              {
                name: "Hunt Count",
                value: `\`${userdata.stats.guilds[message.channel.guild.id].huntCount}\``,
                inline: true
              },
              {
                name: "Battle Count",
                value: `\`${userdata.stats.guilds[message.channel.guild.id].battleCount}\``,
                inline: true
              },
              {
                name: "Pray/Curse Count",
                value: `\`${userdata.stats.guilds[message.channel.guild.id].praycurseCount}\``,
                inline: true
              },
              {
                name: "\u200B",
                value: "\u200B",
                inline: false
              },
              {
                name: "Daily OwO Count",
                value: `\`${userdata.stats.guilds[message.channel.guild.id].dailyOwoCount}\``,
                inline: true
              },
              {
                name: "Daily Hunt Count",
                value: `\`${userdata.stats.guilds[message.channel.guild.id].dailyHuntCount}\``,
                inline: true
              },
              {
                name: "Daily Battle Count",
                value: `\`${userdata.stats.guilds[message.channel.guild.id].dailyBattleCount}\``,
                inline: true
              },
              {
                name: "Daily Pray/Curse Count",
                value: `\`${userdata.stats.guilds[message.channel.guild.id].dailyPraycurseCount}\``,
                inline: true
              }
            ],
            timestamp: new Date()
          }
        }
        bot.createMessage(message.channel.id, statsEmbed)
      }

    })
  }, {
    cooldown: 3000,
    cooldownMessage: "Whoa there slow down, the cooldown is 3 seconds!"
  })
}

File: src/main/java/dastrangeboi/atlantis/Commands/Serverstats.java
package dastrangeboi.atlantis.Commands;

import dastrangeboi.atlantis.Database;
import net.dv8tion.jda.api.EmbedBuilder;
import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
import org.bson.Document;

public class Serverstats {

    private static String getStats(Document userdata, String statName, String guildID) {
        Document stats = (Document) userdata.get("stats");
        Document guildStats = (Document) stats.get("guilds");
        Document serverStats = (Document) guildStats.get(guildID);
        return "`"+serverStats.getInteger(statName)+"`";
    }

    public static void run(GuildMessageReceivedEvent message) {
        Document userdata = Database.getUserdata(message.getAuthor().getId());
        String guildID = message.getGuild().getId();
        EmbedBuilder statsEmbed = new EmbedBuilder()
                .setAuthor(message.getAuthor().getName()+"'s stats in " + message.getGuild().getName(), null, message.getAuthor().getAvatarUrl())
                .setColor(message.getGuild().getSelfMember().getColor())
                .setThumbnail(message.getGuild().getIconUrl())
                .addField("OwO Count", getStats(userdata, "owoCount", guildID), true)
                .addField("Hunt Count", getStats(userdata, "huntCount", guildID), true)
                .addField("Battle Count", getStats(userdata, "battleCount", guildID), true)
                .addField("Pray/Curse Count", getStats(userdata, "praycurseCount", guildID), true)
                .addBlankField(false)
                .addField("Daily OwO Count", getStats(userdata, "dailyOwoCount", guildID), true)
                .addField("Daily Hunt Count", getStats(userdata, "dailyHuntCount", guildID), true)
                .addField("Daily Battle Count", getStats(userdata, "dailyBattleCount", guildID), true)
                .addField("Daily Pray/Curse Count", getStats(userdata, "dailyPraycurseCount", guildID), true);

        message.getChannel().sendMessage(statsEmbed.build()).queue();
    }
}
